internal/native/client.go

```diff
@@ -323,6 +323,61 @@ func (c *Client) PutContainer(params map[string]string) PutContainerResponse {
 	return PutContainerResponse{Success: true, ContainerID: contID.EncodeToString()}
 }
 
+type filter struct {
+	Key       string
+	Operation string
+	Value     string
+}
+
+func (c *Client) Search(cnrString string, filtersJS []filter) (int, error) {
```

Comment:
why `filter` isnt exported?

Reply:
why it should be? it is only a container that will be used when code is translated from JS to go

Reply:
why method is exported then? im lookin at this code as a regular Go one

Reply:
> why method is exported then?

to make it possible to be called in k6 code

> im lookin at this code as a regular Go one

idk, i think this code should be considered as a private package code, it is called by an external k6 code that is never human (and that is never written based on our neofs code) and that just looks for exported fields with appropriate names. can be exported, OK, but changes nothing